scene/gui/label.cpp

```diff
@@ -36,6 +36,34 @@
 
 #include "servers/text_server.h"
 
+Ref<Font> Label::_get_font() const {
+	switch (header_mode) {
+		case HEADER_SMALL:
+			return get_theme_font("font_header_small");
+		case HEADER_MEDIUM:
+			return get_theme_font("font_header_medium");
+		case HEADER_LARGE:
+			return get_theme_font("font_header_large");
+		default:
+			get_theme_font("font_normal");
+	}
+	return Ref<Font>();
```

Comment:
It should probably be just `return get_theme_font("font_normal");` either in the `default` case or outside of the `switch`.